extension.json

```diff
@@ -22,6 +23,10 @@
       {
         "title": "Open on GitHub",
         "command": "github-tools.openFile"
+      },
+      {
+        "title": "Copy GitHub Reference to Clipboard",
```

Comment:
Thanks! Do you mean to copy the github URL or just the ref (e.g. commit sha, branch, etc.)? If it's URL, as the code appears to do, can we use "GitHub URL" instead of "GitHub Reference"?